src/com/cardstore/controller/admin/ShowReviewManageServlet.java

```diff
@@ -0,0 +1,61 @@
+package com.cardstore.controller.admin;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.cardstore.dao.ReviewDAO;
+import com.cardstore.entity.Review;
+
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ * @author Sera Jeong 12211242 Created Date: 24/08/2024
+ */
+
+@WebServlet("/admin/review_manage")
+public class ShowReviewManageServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	public ShowReviewManageServlet() {
+		super();
+	}
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		
+
+        ReviewDAO reviewDAO = new ReviewDAO();
```

Comment:
Servlet should not work with DAO directly, but need to go through Service. We should handle them in ReviewServices class instead of Servlet